melatonin_inspector.h

```diff
@@ -66,21 +66,77 @@ namespace melatonin
             inspectorComponent.setLookAndFeel (&inspectorLookAndFeel);
 
             setResizable (true, false);
-
+            setSettings (settings_);
             updateWindowSize();
 
             setUsingNativeTitleBar (true);
             setContentNonOwned (&inspectorComponent, true);
             setupCallbacks();
         }
 
+        void setSettings (juce::PropertiesFile* settings_)
+        {
+            if (settings_ != nullptr)
+            {
+                settings.set (settings_, false);
+            }
+            else
+            {
+                juce::PropertiesFile::Options opts;
+
+                opts.applicationName = "refx";
```

Comment:
should we change this to "melatonin_inspector"?

Reply:
Yes, oops. Copy / pasted from some other code